### PR TITLE
Update Jekyll & more

### PR DESCRIPTION
**Pull request by dancork**
#### What does this PR do? (please provide any background)
Well! GitHub pages has it's own gem that helps you update it easier. The latest version of jekyll has a shedload of features too, including sass support.

What's change:
- Changed to using `github-pages` gem
- LESS converted to SASS ( built in to jekyll )
- Changed code hightlighter from js to using Rouge ( built in to jekyll )
- Improved build scripts
- General cleanup

#### What tests does this PR have?
N/a

#### How can this be tested?
- Remove node modules `rm -rf node_modules/`
- Optional remove jekyll `gem uninstall jekyll` and select all versions
- Install all using single script `npm i`
- Run locally `npm start`

Hopefully very little should look different. The only UI change is the syntax highlighter now uses rouge.

#### What gif best describes how you feel about this work?

---

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

**Comment by connormeredith** [comment]
Taking review :point_up:

**Comment by jodiedoubleday** [comment]
At the moment the whole company uses LESS, and pattern library/brands all use LESS as well. We are in the process of creating a Hungry Geek brand in pattern library. So if possible i would prefer to keep this in keeping with the rest of our systems for now.

Happy for the res t of the upgrade.

File: README.md [hunk 1]
```diff
@@ -81,27 +81,14 @@ It uses HTML files for structure and Markdown files for content.
...
-npm run watch     # watches for changes to less/js
-
+npm i             # Installs all node modules and ruby gems
 ```
 
 #### To Run
 
 ```
-npm start         # this will compile, minify, watch and start jekyll
+npm start         # this will compiles, start jekyll and watch for changes
```
Comment (connormeredith) [inline README.md]: >compiles

**Comment by dancork** [comment]
@jodiedoubleday I did consider this but at the same time HG is fairly standalone and nothing else uses this tech. For the sake of massively simplifying and cleaning up the codebase I feel SASS is the way to go with this.

If we decide to move HG to metalsmith, or similar, and start implementing the toolkit/brands then yes LESS makes sense and we can easily convert it backwards with a script ( i used less2sass and it took approx 3 secs ).

File: _posts/2016-01-04-thoughts-on-nightwatch.md [hunk 2]
```diff
@@ -21,12 +21,13 @@ Nightwatch has excellent documentation from installation to an API reference. I'
 Found here: [Nightwatch guide](http://nightwatchjs.org/guide) - [Nightwatch Api](http://nightwatchjs.org/api)
 
 ### Clean syntax
-Nightwatch has clear syntax meaning it’s easier for people to understand a written test (or another person’s). This is simple but powerful as tests are neatly laid out and make for a clean test. <br/> Example: 
-<pre><code>
+Nightwatch has clear syntax meaning it’s easier for people to understand a written test (or another person’s). This is simple but powerful as tests are neatly laid out and make for a clean test. <br/> Example:
```
Comment (connormeredith) [inline _posts/2016-01-04-thoughts-on-nightwatch.md]: Can we remove the `<br/>` from here?

Comment (dancork) [inline _posts/2016-01-04-thoughts-on-nightwatch.md]: i'm going to put in a separate PR to cleanup html/markdown :)

Comment (connormeredith) [inline _posts/2016-01-04-thoughts-on-nightwatch.md]: Cool

File: _sass/syntax.scss [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  .sx { color: #d01040 } /* Literal.String.Other */
+  .sr { color: #009926 } /* Literal.String.Regex */
+  .s1 { color: #d01040 } /* Literal.String.Single */
+  .ss { color: #990073 } /* Literal.String.Symbol */
+  .bp { color: #999999 } /* Name.Builtin.Pseudo */
+  .vc { color: #008080 } /* Name.Variable.Class */
+  .vg { color: #008080 } /* Name.Variable.Global */
+  .vi { color: #008080 } /* Name.Variable.Instance */
+  .il { color: #009999 } /* Literal.Number.Integer.Long */
+
```
Comment (connormeredith) [inline _sass/syntax.scss]: Extra line here :wink:

**Comment by connormeredith** [comment]
:+1: Great work dude

**Comment by jackdcrawford** [comment]
le :two: :mag:

**Comment by jackdcrawford** [comment]
:+1:
